config/initializers/assets.rb

```diff
@@ -9,3 +9,4 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
+Rails.application.config.assets.precompile += %w( email.css )
```

Comment:
why exactly is this needed? (i dont end up having to do this often so just curious)